src/app/pipes/content-filter.pipe.ts

```diff
@@ -0,0 +1,24 @@
+import { Pipe, PipeTransform } from '@angular/core';
+import { Link } from '../models/Link';
+
+@Pipe({
+    name: 'contentFilter'
+})
+
+export class ContentFilterPipe implements PipeTransform {
```

Comment:
Where is this filter used?

Reply:
Feature I was going to implement but deemed to complicated. Deleted in the next commit